roxctl/declarativeconfig/create/permission_set.go

```diff
@@ -0,0 +1,83 @@
+package create
+
+import (
+	"sort"
+	"strings"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/declarativeconfig"
+	"github.com/stackrox/rox/pkg/errox"
+	"github.com/stackrox/rox/pkg/maputil"
+	"github.com/stackrox/rox/roxctl/common/environment"
+	"gopkg.in/yaml.v3"
+)
+
+func permissionSetCommand(cliEnvironment environment.Environment) *cobra.Command {
+	permSetCmd := &permissionSetCmd{permissionSet: &declarativeconfig.PermissionSet{}, env: cliEnvironment}
+
+	cmd := &cobra.Command{
+		Use:  permSetCmd.permissionSet.Type(),
+		Args: cobra.NoArgs,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if err := permSetCmd.Validate(); err != nil {
+				return errors.Wrap(err, "validating the permission set")
+			}
+			return permSetCmd.PrintYAML()
+		},
+		Short: "Create a declarative configuration for a permission set",
+	}
+
+	cmd.Flags().StringVar(&permSetCmd.permissionSet.Name, "name", "", "name of the permission set")
+	cmd.Flags().StringVar(&permSetCmd.permissionSet.Description, "description", "",
+		"description of the permission set")
+	cmd.Flags().StringToStringVar(&permSetCmd.resourceWithAccess, "resource-with-access", map[string]string{},
+		`list of resources with the respective access, e.g. --resource-with-access Access=READ_ACCESS,Admin=READ_WRITE_ACCESS
```

Comment:
```suggestion
		`list of resources with the respective access, e.g. --resource-with-access Access=READ_ACCESS,Administration=READ_WRITE_ACCESS
```

There is no such resource as `Admin`. Let's have only valid resources as examples